src/mscorlib/src/System/Diagnostics/Eventing/EventSource.cs

```diff
@@ -5804,6 +5828,74 @@ public enum EventManifestOptions
         AllowEventSourceOverride = 0x8,
     }
 
+#if FEATURE_EVENTSOURCE_XPLAT 
+
+    internal  class XplatEventLogger : EventListener
```

Comment:
Why would there be a different plan ? Why can't the existing plan be adapted to it ?